plugins/woocommerce-admin/client/products/fields/attribute-input-field/attribute-input-field.tsx

```diff
@@ -46,41 +50,46 @@ export const AttributeInputField: React.FC< AttributeInputFieldProps > = ( {
 	} );
 
 	const getFilteredItems = (
-		allItems: Pick< QueryProductAttribute, 'id' | 'name' >[],
+		allItems: NarrowedQueryAttribute[],
 		inputValue: string
 	) => {
-		return allItems.filter(
-			( item ) =>
-				filteredAttributeIds.indexOf( item.id ) < 0 &&
+		const onlyIdsFilter = ( item: NarrowedQueryAttribute ) =>
```

Comment:
I believe this filter seems to break the select control, where I cannot search for any other attribute.

Reply:
Do you mean in just in the edit modal? That's my intention here and what I assumed would be the behavior we want (unless I misunderstand). See my other comment above for more context.

Reply:
I finally got it working, it may of been a couple things. The need for the use of the `SelectControlMenuSlot` to have the dropdown show correctly and I was testing this on a site that had over a 1000 of attributes, so the selected items may of not come back as part of the initial list, thus not showing up, also reasoning for just using the standard `SelectControl` as mentioned above.